ansible_workspace_dir/roles/protocol-server-client/tasks/create-and-save-env-file.yml

```diff
@@ -0,0 +1,9 @@
+---
+  - name: Ensure env directory exists
```

Comment:
A role to create and add default.yaml needs to be added as well. Can you add that as well ?